messages/subjects.yaml

```
@@ -35,11 +37,13 @@ rate_of_turn_degpm:                     keelson.TimestampedFloat
 
 heading_true_north_deg:                 keelson.TimestampedFloat
 heading_magnetic_deg:                   keelson.TimestampedFloat
+headding_accuracy_deg:                  keelson.TimestampedFloat
```

```suggestion
heading_accuracy_deg:                  keelson.TimestampedFloat
```

What does this mean? What does a value of 2 mean here?